src/main/java/org/opentripplanner/routing/alternativelegs/AlternativeLegs.java

```diff
@@ -45,18 +45,38 @@ public static List<ScheduledTransitLeg> getAlternativeLegs(
     TransitService transitService,
     boolean searchBackward,
     AlternativeLegsFilter filter
+  ) {
+    return getAlternativeLegs(
+      leg,
+      numberLegs,
+      transitService,
+      searchBackward,
+      filter,
+      false,
+      false
+    );
+  }
+
+  public static List<ScheduledTransitLeg> getAlternativeLegs(
+    Leg leg,
+    Integer numberLegs,
+    TransitService transitService,
+    boolean searchBackward,
+    AlternativeLegsFilter filter,
+    boolean exactOriginStop,
+    boolean exactDestinationStop
```

Comment:
Is there a use case where these would differ?

Reply:
Someone might want to only use exact stop for the origin stop as it's slightly more confusing for users if they need to switch from which platform they depart from if it changes between departures but it might not matter to which stop they arrive to within a terminal.